docs/release-notes/v1.0.1.rst

```diff
@@ -0,0 +1,54 @@
+.. include:: ../references.txt
+
+.. _gammapy_1p0p1_release:
+
+1.0.1 (March 14th, 2023)
+------------------------
+
+Summary
+~~~~~~~
+
+- Released Mar 14th, 2023
+- 9 contributors
+- 31 pull requests since v1.0 (not all listed below)
+
+This is the first bug-fix after v1.0. Several minor bugs and typos in the documentation are corrected.
+
+- Few corrections have been made to improve the performance of the models evaluation in particular for 1d 
+  spectral analyses. The performances of benchmarks have significantly improved.
```

Comment:
```suggestion
  spectral analyses.
```
Would just remove the sentence, only mention the effect to users as in the sentence before.